userd/ui/kdialog.go

```diff
@@ -0,0 +1,65 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package ui
+
+import (
+	"fmt"
+	"os/exec"
+	"time"
+)
+
+type Kdialog struct{}
+
+func (*Kdialog) YesNo(primary, secondary string, options *Options) bool {
+	if options == nil {
+		options = &Options{}
+	}
+
+	txt := fmt.Sprintf(`<p><big><b>%s</b></big></p><p>%s</p>`, primary, secondary)
```

Comment:
Not a practical risk here but if `primary` or `secondary` contain any formatting code it could lead to incorrect rendering.